infor-design/enterprise
# 518 - Fix icon pipeline

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This issue fixes the icon pipeline for changes made where the stroke is `000#`. Also made the files downloadable.
To do this needed to add some css in both ends of the pipeline and other adjustments.

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/design-system/issues/518

**Steps necessary to review your pull request (required)**:
- make sure to do `npm i` to get the new packages
- go to http://localhost:4000/components/icons
- test all these icon pages
- make sure to change themes and see the colors change on the icons
- open up this folder node_modules/ids-identity/dist/theme-new/icons/standard/svg
- and try to open the svg files in the browser. This should work now
- smoke test http://localhost:4000

**Included in this Pull Request**:
